src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs

```diff
@@ -91,6 +91,11 @@ public sealed class Features
         /// </summary>
         internal bool IsLookUpReductionDelegationEnabled { get; set; }
 
+        /// <summary>
+        /// Enables User-defined types functionality.
+        /// </summary>
+        internal bool IsUserDefinedTypesEnabled { get; set; } = false;
```

Comment:
>IsUserDefinedTypesEnabled [](http://example.com/codeflow?start=22&length=25)

We probably want to enable it for the PowerFxV1 property below - this is what is used on the C# interpreter; we want to have it enabled by default when we start supporting it.